### DISCUSSION TITLE
Zotero integration not working?

### DISCUSSION THREAD

**Discussion by yaxu** [knowledgefutures/pubpub#2771] [Support]
I'd love to be able to add citations from zotero! When I click the 'zotero integration' button, it takes me to a privacy page. I click 'activate' under the zotero section, nothing happens for a while, and then it times out with an error page. 

![image](https://github.com/pubpub/pubpub/assets/27940/25caffa3-f6e8-46bb-a836-db9d0f37b4f2)

**Comment by gabestein** [comment]
Thanks for reporting! We've got a fix in the works that should be up today.
